types/time_test.go

```diff
@@ -122,6 +122,7 @@ func (s *testTimeSuite) TestDateTime(c *C) {
 		{"2017-01-05 23:59:59.575601", 0, "2017-01-06 00:00:00"},
 		{"2017-01-31 23:59:59.575601", 0, "2017-02-01 00:00:00"},
 		{"2017-00-05 23:59:58.575601", 3, "2017-00-05 23:59:58.576"},
+		{"2020-05-19T17:51:57.138597+09:00", 6, "2020-05-19 16:51:57.138597"},
```

Comment:
Thanks for your contribution @xiaodong-ji .
Could you please add more tests about DateTime values with timezone here?

Reply:
OK，let me add more tests.

Reply:
@qw4990